lib/search.ts

```diff
@@ -0,0 +1,101 @@
+import {
+  GetPackageData,
+  PackageResource,
+  MavenPackage,
+  SnykHttpClient,
+} from './parse/types';
+import { PackageURL } from 'packageurl-js';
+import * as debugLib from 'debug';
+import * as path from 'path';
+
+const debug = debugLib('snyk-mvn-plugin');
+
+const PACKAGE_SEARCH_TYPE = 'maven';
+const PACKAGE_SEARCH_ENDPOINT = '/rest/packages';
```

Comment:
we don't need `/rest` here - the http client prefixes that for us

Reply:
see https://github.com/snyk/snyk-gradle-plugin/pull/238/files where we made the same mistake